changelog_checklist_formatter/spec/changelog_jira_formatter_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rspec'
+require_relative '../lib/changelog_checklist'
+
+describe 'changelog formatter' do
+  let(:version_series) { '' }
+  let(:formatter) { ChangelogChecklist.new(version_series) }
+
+  context 'when the version series is 4.63.0' do
+    let(:version_series) { '4.63.0' }
+    let(:output1) {
+      "|4.63.0| |\n"\
+      "|*Internal*| |\n"\
+      "|[TT-569] Use the tagged base image to improve traceability| |\n"
+    }
+    it 'the ChangeLog table will be formatted as output1' do
+      expect(formatter.to_table).to eq output1
```

Comment:
where is the formatter been given the output 1 input? It seems to read the file from a given path via the CHANGELOG_PATH environment variable. 

If the specs are passing for you, it's possibly because these outputs are based on the real changelog from QT.

The examples should probably not be real entries, but just some mock ones that cover the required test cases in terms of versions and sections.